MonoGame.Framework/Platform/Graphics/RenderTarget2D.DirectX.cs

```diff
@@ -13,14 +13,12 @@ public partial class RenderTarget2D
         internal DepthStencilView _depthStencilView;
         private SharpDX.Direct3D11.Texture2D _msTexture;
 
-        private SampleDescription _sampleDescription;
+        private SampleDescription _msSampleDescription;
```

Comment:
rename field to _msSampleDescription to avoid mixup with base._sampleDescription.